modules/shared/logic/avatarLogic/avatarLogic.ts

```diff
@@ -0,0 +1,12 @@
+export const determineAvatarSize = (componentSize: string): number => {
```

Comment:
## Enhancement 
you can add a function description, function parameters, return value. `documentation comments`
- hovering on the function name will show a documentation popup with that comment.
- it helps in code maintenance also we can 
- we may later discuss and use documentation generator packages such as [jsDocs](https://github.com/jsdoc/jsdoc). we may discuss it later not now 😂 (no need to waste time or to explore it) just to keep in mind.
```
/**
 * This function accepts a string as a parameter and return a text transformed to lowercase letters
 * @param text - The function parameter
 * @returns Returns a text tranformed to uppercase
 */

const textToLowerCase = (text: string): string => {
  return text.toLowerCase();
};

export default textToLowerCase;

```